src/domain/index.ts

```diff
@@ -1,15 +1,41 @@
 import { BigNumber } from "ethers";
-import { FC } from "react";
+import { ComponentType } from "react";
+
+export enum SupportedNetwork {
+  LOCAL = "local",
+}
+
+export interface Chain {
+  name: string;
+  chainId: number;
+  Icon: ComponentType<{ className?: string }>;
+}
+
+export interface Token {
+  name: string;
+  symbol: string;
+  address: string;
+  decimals: number;
+  logoURI: string;
+  chainId: number;
+}
+
+export interface BridgeConfig {
+  l1ContractAddress: string;
+  l2ContractAddress: string;
+  apiUrl: string;
+}
 
 export interface Env {
-  REACT_APP_INFURA_API_KEY: string;
-  REACT_APP_L1_PROVIDER_NETWORK: string;
-  REACT_APP_L2_PROVIDER_URL: string;
-  REACT_APP_BRIDGE_CONTRACT_ADDRESS: string;
-  REACT_APP_UNISWAP_QUOTER_CONTRACT_ADDRESS: string;
-  REACT_APP_USDT_CONTRACT_ADDRESS: string;
-  REACT_APP_FIAT_EXCHANGE_RATES_API_KEY: string;
-  REACT_APP_BRIDGE_API_URL: string;
+  fiatExchangeRatesApiKey: string;
+  infuraApiKey: string;
+  bridgeConfig: BridgeConfig;
+  chains: Chain[];
+  l1ProviderUrl: string;
+  l2ProviderUrl: string;
+  uniswapQuoterAddress: string;
+  ethToken: Token;
+  usdtToken: Token;
```

Comment:
This interface no longer represents the `.env` but a `Network` (as commented before).